lib/membrane/video_compositor/handler/callback_ctx.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Membrane.VideoCompositor.Handler.CallbackContext do
+  @moduledoc """
+  Structure representing a common part of the context
+  for all of the callbacks.
+  """
+  alias Membrane.{Pad, Time}
+  alias Membrane.VideoCompositor.Scene
+
+  defmacro __using__(custom_fields) do
+    default_fields =
+      quote do
+        [
+          input_pads: list(Pad.ref_t()),
```

Comment:
`input_pads` is a little bit ambiguous when it comes to `handle_init`, `handle_video_add`, and `handle_video_remove` callbacks. 

IMO there isn't one obvious answer to those questions:
1.  Does `input_pads` in `Membrane.VideoCompositor.Handler.CallbackContext.VideoAdd` contains just added video?
2. Does `input_pads` in `Membrane.VideoCompositor.Handler.CallbackContext.VideoRemove` contains the just removed video?
3. What it contains in `Membrane.VideoCompositor.Handler.CallbackContext.Init`?

So many questions, so few answers...

Maybe leaving just a note somewhere in the docs, that this should e.x. contain